Project/TwoPlayerGame_Project.uvprojx

```diff
@@ -461,14 +446,44 @@
               <FilePath>.\Source\system_stm32f4xx.c</FilePath>
             </File>
             <File>
-              <FileName>libraryProject.lib</FileName>
-              <FileType>4</FileType>
-              <FilePath>..\Libraries\libraryProject.lib</FilePath>
+              <FileName>stm32f4xx_exti.c</FileName>
+              <FileType>1</FileType>
+              <FilePath>..\Libraries\STM32F4xx_StdPeriph_Driver\src\stm32f4xx_exti.c</FilePath>
+            </File>
+            <File>
+              <FileName>stm32f4xx_gpio.c</FileName>
+              <FileType>1</FileType>
+              <FilePath>..\Libraries\STM32F4xx_StdPeriph_Driver\src\stm32f4xx_gpio.c</FilePath>
+            </File>
+            <File>
+              <FileName>stm32f4xx_rcc.c</FileName>
+              <FileType>1</FileType>
+              <FilePath>..\Libraries\STM32F4xx_StdPeriph_Driver\src\stm32f4xx_rcc.c</FilePath>
+            </File>
+            <File>
+              <FileName>misc.c</FileName>
+              <FileType>1</FileType>
+              <FilePath>..\Libraries\STM32F4xx_StdPeriph_Driver\src\misc.c</FilePath>
+            </File>
+            <File>
+              <FileName>stm32f4xx_syscfg.c</FileName>
+              <FileType>1</FileType>
+              <FilePath>..\Libraries\STM32F4xx_StdPeriph_Driver\src\stm32f4xx_syscfg.c</FilePath>
+            </File>
+            <File>
+              <FileName>stm32f4xx_dma.c</FileName>
+              <FileType>1</FileType>
+              <FilePath>..\Libraries\STM32F4xx_StdPeriph_Driver\src\stm32f4xx_dma.c</FilePath>
+            </File>
+            <File>
+              <FileName>stm32f4xx_i2c.c</FileName>
+              <FileType>1</FileType>
+              <FilePath>..\Libraries\STM32F4xx_StdPeriph_Driver\src\stm32f4xx_i2c.c</FilePath>
             </File>
             <File>
-              <FileName>stm32f4xx_fmc.c</FileName>
+              <FileName>stm32f4xx_spi.c</FileName>
               <FileType>1</FileType>
-              <FilePath>..\Libraries\STM32F4xx_StdPeriph_Driver\src\stm32f4xx_fmc.c</FilePath>
+              <FilePath>..\Libraries\STM32F4xx_StdPeriph_Driver\src\stm32f4xx_spi.c</FilePath>
             </File>
           </Files>
         </Group>
```

Comment:
Tek nakon sta sam pritisnia approve sam ovo vidia, nebi trebala mijenjat strukturu projekta ovoliko jer ako se ovo merga prestat ce radit kod koji je vec u developu. Trebala bi ostat ista struktura + uključeni tvoji fileovi gyro.c, gyro.h i test.c. (Ne trebas ukljucivat sve ove drivere iz library-a jer su oni svi ukljuceni kroz .lib file, pogledaj u readme projekta)
Usput kad vec to ides ispravljat ponisti i promjene na gitignoreu i na main.h.

Ovako bi to trebalo izgledat (+ 3 nova filea):
![capture](https://user-images.githubusercontent.com/33004384/35008819-20736754-fafe-11e7-82f6-8cdd18376e24.PNG)